recipes/poco/all/conanfile.py

```diff
@@ -190,6 +190,7 @@ def package_info(self):
                 self.cpp_info.libs.append("%s%s" % (lib, suffix))
 
         self.cpp_info.libs.append("PocoFoundation%s" % suffix)
+        self.cpp_info.libs.append("PocoEncodings%s" % suffix)
```

Comment:
since when Encodings is mandatory? IIRC only PocoFoundation is available by default. Does it require an option?

Reply:
emmm... Sorry,I see that this option exists, is it correct to add it here?
``` python
def package_info(self):
    libs = [("enable_mongodb", "PocoMongoDB"),
            ("enable_pdf", "PocoPDF"),
            ("enable_netssl", "PocoNetSSL"),
            ("enable_netssl_win", "PocoNetSSLWin"),
            ("enable_net", "PocoNet"),
            ("enable_crypto", "PocoCrypto"),
            ("enable_data_sqlite", "PocoDataSQLite"),
            ("enable_data_mysql", "PocoDataMySQL"),
            ("enable_data_odbc", "PocoDataODBC"),
            ("enable_data", "PocoData"),
            ("enable_sevenzip", "PocoSevenZip"),
            ("enable_zip", "PocoZip"),
            ("enable_apacheconnector", "PocoApacheConnector"),
            ("enable_util", "PocoUtil"),
            ("enable_xml", "PocoXML"),
            ("enable_json", "PocoJSON"),
            ("enable_redis", "PocoRedis"),
            ("enable_encodings", "PocoEncodings")] #Add PocoEncodings
```